plat/hisilicon/hikey960/aarch64/hikey960_common.c

```diff
@@ -41,7 +41,7 @@
 					TSP_SEC_MEM_SIZE,		\
 					MT_MEMORY | MT_RW | MT_SECURE)
 
-#if LOAD_IMAGE_V2
+#if LOAD_IMAGE_V2 && (HIKEY960_TSP_RAM_LOCATION_ID == HIKEY960_SRAM_ID)
```

Comment:
`HIKEY960_OPTEE_PAGEABLE_LOAD_*` are not defined for SRAM use atm, so maybe just remove `MAP_OPTEE_PAGEABLE` from `hikey960_mmap` until they're defined for SRAM use?

Same for below.

Reply:
OK. I'll remove it.